### PR TITLE
Feature/issues for asset type

### PR DESCRIPTION
**Pull request by lsantire**
## Changes proposal

- Changed the logic to get issues list of an specific asset, in order to only retrieve issues of alerts that correlate with the asset type.

## Testing

Local manual testing.

### CODE REVIEW

File: iot_api/user_api/resources/Inventory.py [hunk 1]
```diff
@@ -270,19 +270,22 @@ def get(self, asset_type, asset_id):
                 until=until,
                 alert_types=[AlertType.find_one(alert_type_code).id for alert_type_code in alert_types],
                 devices=[asset.id],
+                alert_asset_type = AlertAssetType.DEVICE,
```
Comment (cyones) [inline iot_api/user_api/resources/Inventory.py]: I think it is better if repository functions receives basic types. Here, alert_asset_type could be a string in ("device", "gateway", None). Inside the function the string is replaced by the corresponding enum type. If the parameter given is not in ("device", "gateway", None) an Error.BadParameter should be thrown.

**Review by cyones — CHANGES_REQUESTED** [review]

File: iot_api/user_api/resources/Inventory.py (hunk 1 above)
Comment (lsantire) [reply iot_api/user_api/resources/Inventory.py]: I agree, done!

File: iot_api/user_api/resources/Inventory.py [hunk 2]
```diff
@@ -156,7 +156,7 @@ def get(self, asset_type, asset_id):
                 order_by=order_by,
                 page=page,
                 size=size,
-                alert_asset_type=AlertAssetType.DEVICE
+                asset_type=asset_type
             )
         else:
```
Comment (cyones) [inline iot_api/user_api/resources/Inventory.py:161]: I think this "if" block could be removed, right?

**Review by cyones — APPROVED** [review]

**Comment by cyones** [comment]
LGTM, there is only an "if" block that I think is useless. If it is the case, delete it and merge the code.

File: iot_api/user_api/resources/Inventory.py (hunk 2 above)
Comment (lsantire) [reply iot_api/user_api/resources/Inventory.py:161]: Not really, the difference between these calls to find_with is that one of them sets the parameter device_id while the other one sets the parameter gateway_id

**Pull request merged**